src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx

```diff
@@ -43,13 +45,14 @@ export const StrategyBlockManage: FC<Props> = ({
   setManage,
   isExplorer,
 }) => {
+  const { strategies } = useStrategyCtx();
```

Comment:
in theory - this makes a single item sensitive to any change to the collection.
I guess that in practice, in our specific case, this is already the case - but I'd look for a way to avoid passing in the strategies collection just so that we can pass it to the event service

Reply:
I'm not sure to understand. Strategies are wrapped in a context, so it's shared no matter how many components listen to it. We are using it in many places in the app already. Using it in this component should have not effect in theory.